backend/climateconnect_main/settings.py

```diff
@@ -131,3 +132,10 @@
 # https://docs.djangoproject.com/en/2.2/howto/static-files/
 
 STATIC_URL = '/static/'
+
+REST_FRAMEWORK = {
+    'DEFAULT_AUTHENTICATION_CLASSES': [
+        'knox.auth.TokenAuthentication',
+        'rest_framework.authentication.SessionAuthentication'
```

Comment:
Does this mean, that session authentication is the fallback?

Reply:
Yes a fallback. Also easy to test APIs locally without frontend integration.